bears/uploadTool.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+# inserting path here so the following imports work wherever you run the script
+# in coala-bears
+sys.path.insert(0, '..')
+
+from coalib.collecting.Importers import iimport_objects
+from string import Template
+
+
+def create_directory(folder_name):
+    """
+    Checks if the folder exists, and creates it if it doesn't.
+
+    :param name: The directory name.
+    """
+    if not os.path.exists(folder_name):
+        os.mkdir(folder_name)
```

Comment:
doesn't mkdir or makedirs have a param that creates it if it's not already there? Then you don't need an own function for it